spec/policies.yml

```diff
@@ -89,8 +89,6 @@ components:
             $ref: 'openapi.yml#/components/responses/Busy'
           "422":
             $ref: 'openapi.yml#/components/responses/UnprocessableEntity'
-          "500":
```

Comment:
when we remove response codes in a PR, can you note the reason why you are doing it?

presumably we added it for a reason, so when we're looking at the git history it would be really nice to know why we remove something like this, too